src/common/config.cc

```diff
@@ -1236,21 +1236,45 @@ bool md_config_t::expand_meta(std::string &origval,
 }
 
 void md_config_t::diff(
+  const md_config_t *other,
+  map<string, pair<string, string> > *diff,
+  set<string> *unknown) 
```

Comment:
@tchaikov , we the same function, with 2 different signatures, where once accepts the 'setting' we are searching for. Would you rather have only 1 function (as it was before) and the optional param for the 'setting' search, which would force us to call it passing a param with ""?